src/config.js

```diff
@@ -7,4 +7,7 @@ module.exports = {
   secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || 'FAKE',
   streamName: process.env.STREAM_NAME,
   listenerAuthToken: process.env.LISTENER_AUTH_TOKEN || 'secret',
+  s3EmailBucket: process.env.S3_EMAILS_BUCKET || 'bucket',
+  s3Endpoint: process.env.S3_ENDPOINT || 'bucketEndpoint',
```

Comment:
I've just been testing this in the demo environment, and I think there's a bug here. In prod, we want `s3Endpoint` to be undefined, so the AWS SDK will default to the normal S3 API. So I think we need to get rid of the ` || 'bucketEndpoint'` here. Does that make sense?